index.js

```diff
@@ -0,0 +1,31 @@
+const langSec = document.querySelector(".language-section");
+const buttons = document.querySelectorAll(".button");
+const textTitle = document.querySelector(".title");
+const textDesc = document.querySelector(".description");
+
+buttons.forEach(button => {
+    button.addEventListener('click', () => {
+        langSec.querySelector(".active").classList.remove(".active");
+        button.classList.add("active");
+
+        const attr = button.getAttribute("data-language");
+
+        textTitle.textContent = data[attr].title;
+        textDesc.textContent = data[attr].description;
+    });
+});
+
+const data = {
```

Comment:
`data` is an incredibly generic name. Whenever possible, always imbue your variable names with meaning that describes the data they hold.

```suggestion
const dictionary = {
```